doc/languages.rst

```diff
@@ -1083,6 +1083,13 @@ to view the docstring of the syntax checker.  Likewise, you may use
 
       Check SQL syntax with `Sqlint <https://github.com/purcell/sqlint>`_.
 
+.. supported-language:: systemd Unit Configuration
+
+   .. syntax-checker:: systemd-analyze
+
+      Check systemd unit configuration file syntax with
+      `systemd-analyze <https://www.freedesktop.org/software/systemd/man/systemd-analyze.html>`_.
```

Comment:
Could you turn this into a "named" link?  e.g.

```
`systemd-analyze`_

.. _systemd-analyze: https://…
```
It's easier to read in plain text form that way.

Don't forget to refill the paragraph afterwards with `M-q`.